MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,387 @@
+---
+layout: post
+title: Grouping in .NET MAUI DataForm control | Syncfusion
```

Comment:
title: Grouping the editors in .NET MAUI DataForm control | Syncfusion

Reply:
changed title